ts/webui/src/components/modals/Compare.tsx

```diff
@@ -218,9 +240,23 @@ class Compare extends React.Component<CompareProps, {}> {
             duration: convertDuration(trial.duration),
             parameters: flatten(trial.parameters(inferredSearchSpace)),
             metrics: flatten(trial.metrics(TRIALS.inferredMetricSpace())),
-            intermediates: _parseIntermediates(trial)
+            intermediates: _parseIntermediates(trial, intermediateKey)
         }));
 
+        if (trials[0].intermediates !== undefined && trials[0].intermediates[0]) {
```

Comment:
What if `trials[0]` has empty intermediates?

Reply:
I think `empty intermediates` is same as `x.intermediates is undefined`...